recipes/bond/all/conanfile.py

```diff
@@ -0,0 +1,57 @@
+from conans import ConanFile, CMake, tools
+import platform, os
+
+
+class BondConan(ConanFile):
+    name = "bond"
+    version = "9.0.0"
+    license = "MIT License"
+    homepage = "https://github.com/microsoft/bond.git"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Bond is a cross-platform framework for working with schematized data. It supports cross-language de/serialization and powerful generic mechanisms for efficiently manipulating data. Bond is broadly used at Microsoft in high scale services."
+    topics = ("bond", "microsoft")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [False, True],
+        "fPIC": [True, False]}
+    default_options = {
+        "shared": False,
+        "fPIC": True}
+    build_requires = "boost/1.71.0@conan/stable"
```

Comment:
```suggestion
    requires = "boost/1.73..0"
```
A `require` is really necessary.
If boost is shared, then the library must be installed too.